### DISCUSSION TITLE
Working with Storybook component library projects

### DISCUSSION THREAD

**Discussion by pgangwani** [pmndrs/jotai#2407] [Q&A]
hello Patrons,
I am having simple to complex fields like dropdown with api call to complex nested form. I would like to know what is the best practice of using / creating stories for such component which involves atom state. May there be dependency between them as derived atom.

Please suggest the best possible approach. So far I have been creating only component work with state. I need to change the strategy.

**Comment by dai-shi** [comment]
Hope someone can help.

If I were to help with something, I would need the problems to be broken down more.
